#slaveNb from 1 to 5
int32 slaveNb
int32 current
---
trs_msgs/MotorCmdSet motorCmdSet
